### PR TITLE
fix: show system message error [AR-3162]

### PR DESCRIPTION
**Pull request by Garzas**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-3162" title="AR-3162" target="_blank">BugAR-3162</a>  PlayTest 21.02 - Ping -> If sending a Ping fails, there should be a warning
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [x] contains a reference JIRA issue number like `SQPIT-764`
    - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Ping is a special regular message which looks like system message. When there is an error when sending ping message, there is no error message to show because error messages was only implemented for regular UI messages.

### Solutions

Implemented UI error also for system messages.

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 
| Before | After |
| ----------- | ------------ |
| before
 | after  |

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1596/1/) **failed**.

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-1596/4/) **failed**.

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-1596/5/) **failed**.

**Comment by AndroidBob** [comment]
Build [6](https://10.10.124.134/job/wire-android-reloaded/job/PR-1596/6/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.1.0(6119585)(6119585)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.1.0(6119585)(6119585)-staging-compat.apk)

File: app/src/main/kotlin/com/wire/android/mapper/MessageMapper.kt [hunk 1]
```diff
@@ -104,19 +104,28 @@ class MessageMapper @Inject constructor(
...
+                        messageContent = content,
+                        source = if (sender is SelfUser) MessageSource.Self else MessageSource.OtherUser,
+                        header = provideMessageHeader(sender, message),
+                        messageFooter = footer,
+                        userAvatarData = getUserAvatarData(sender)
+                    )
+                }
+            }
+            is UIMessageContent.SystemMessage ->
+                UIMessage.System(
```
Comment (saleniuk) [inline app/src/main/kotlin/com/wire/android/mapper/MessageMapper.kt:122]: I think system message can also have `message.visibility == HIDDEN`, right? It's missed here, previous code was including this case 😉

**Pull request closed without merging**